src/Microsoft.AspNet.Mvc.ViewFeatures/ViewFeatures/DefaultHtmlGenerator.cs

```diff
@@ -106,6 +107,7 @@ public string FormatValue(object value, string format)
 
         /// <inheritdoc />
         public virtual TagBuilder GenerateActionLink(
+            ViewContext viewContext,
```

Comment:
I don't see a reason to change this. A lot of stuff has access to view context here. It seems like really **all** of these should from a design POV.